integration_tests/backend/setup/changing_saves.py

```diff
@@ -19,7 +19,7 @@
 
 table = integration.sql(query="SELECT * FROM wine;")
 
-table.save(integration.config(table="table_1", update_mode="append"))
+table.save(integration.config(table="table_1", update_mode="replace"))
```

Comment:
Update this so we aren't appending to a non-existing table and thus these flows fail